great_expectations/checkpoint/checkpoint.py

```diff
@@ -931,7 +937,7 @@ def construct_from_config_args(
         data_context: AbstractDataContext,
         checkpoint_store_name: str,
         name: str,
-        config_version: Union[int, float] = 1.0,
+        config_version: Optional[Union[int, float]] = None,
```

Comment:
Why make this change? I think we should be keeping it `1.0` since a regular `Checkpoint` is far more widely used than a `LegacyCheckpoint`

Reply:
Fixed.  Thanks!